src/nimblepkg/packageparser.nim

```diff
@@ -258,10 +258,16 @@ proc readPackageInfoFromNimble(path: string; result: var PackageInfo) =
             for i in ev.value.multiSplit:
               if i.splitFile().ext == ".nim":
                 raise newException(NimbleError, "`bin` entry should not be a source file: " & i)
+              let (binSrc, bin) = if '=' notin i: (i, i) else:
+                let spl = i.split('=', 1)
+                (spl[0], spl[1])
```

Comment:
I don't like this. I think you should introduce a new field called `namedBin` or `aliasedBin` or something and use Nim's tuple syntax. Don't make up even more arbitrary parsing rules(!)

Reply:
I don't like it either but did this as an example - it results in minimal code changes. Adding a new field will be more code both in implementation (need to get the data from `nim e` back into nimble) as well as for users.

```nim
bin = @["test1"]
namedBin = @[("test2", "test-2")]
```

vs.

```nim
bin = @["test1", "test2=test-2"]
```

There's also the following example if we use tables:

```nim
bin = @["test1"]
namedBin["test2"] = "test-2"
```

Reply:
Okay, I could get `namedBin: Table[string, string]` for really cheap - data is still forwarded as `test2=test-2` in `bin` but the user doesn't need to see that. They can use either of:

```nim
namedBin["test2"] = "test-2"
namedBin = {"test2": "test-2"}.toTable
```